ONSdigital/srm-rh-ui
# 1078 Set up gunicorn runner

# Motivation and Context

We need to run the flask app on a production grade server. We have chosen Gunicorn initially as it is a well proven and documented server for flask apps.

# What has changed



- Add gunicorn config
- Run with gunicorn in docker container
- Configure the gunicorn arbiter and access logs
- Remove now unnecessary application access logs

# How to test?



Build and run locally, check the server still behaves as expected. The Flask based `run.py` should still work as previously, enabling local development and debugging running on the Flask server, but the docker image must now always run on the prod grade gunicorn server

# Links



https://trello.com/c/MFpmW5we/1078-rh-ui-set-up-gunicorn-runner-8
https://trello.com/c/9QpQxMbj/1089-rh-ui-review-and-standardize-logging5
